pages/reply.js

```diff
@@ -78,10 +75,8 @@ class ReplyPage extends React.Component {
       : '查看文章頁面';
 
     return (
-      <Link
-        href={`/article?id=${originalArticle.get('id')}`}
-        as={`/article/${originalArticle.get('id')}`}
-      >
+      <Link route='article' params={{id: article.get('id')}}>
```

Comment:
Running `yarn lint` should tell you that `article` is not defined. Please help fix this.